acceptance/tests/partitions/partitions_test.go

```diff
@@ -258,7 +251,6 @@ func TestPartitions(t *testing.T) {
 			// Ensure consul client are created.
 			agentPodList, err := clientClusterContext.KubernetesClient(t).CoreV1().Pods(clientClusterContext.KubectlOptions(t).Namespace).List(ctx, metav1.ListOptions{LabelSelector: "app=consul,component=client"})
 			require.NoError(t, err)
-			require.Len(t, agentPodList.Items, 1)
```

Comment:
In a cloud env, there will be more than one client agent bc there will be multiple nodes. I could add back this check just for Kind but it seemed not essential

Reply:
i agree..it's not essential!!